packages/fiori/src/SideNavigationSelectableItemBase.ts

```diff
@@ -178,6 +178,10 @@ class SideNavigationSelectableItemBase extends SideNavigationItemBase {
 			classes.push("ui5-sn-item-selected");
 		}
 
+		if (this.unselectable) {
+			classes.push("ui5-sn-item-unselectable");
+		}
```

Comment:
why don't you select by the host, for instance `:host([unselectable]) ... `
This will save the use of this extra class

Reply:
Rather `:host([unselectable]:not([side-nav-collapsed])) .ui5-sn-item-toggle-icon`

Reply:
OK